hvpy/conftest.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pytest
+
+
+class SetEnv:
+    def __init__(self):
+        self.envars = set()
+
+    def set(self, name, value):
+        self.envars.add(name)
+        os.environ[name] = value
+
+    def clear(self):
+        for n in self.envars:
+            os.environ.pop(n)
```

Comment:
Should you reset self.envars back to an empty set after this?

Reply:
I think it would be safe to do in a unit test but I can make this take a list of keys to remove instead?

Reply:
Wait, I can't read code. This will only remove envs that have been added by this class.
So doing this back to an empty set is ok, since it won't affect anything existing ones in `os.environ`